torch/distributed/nn/api/remote_module.py

```diff
@@ -170,14 +172,18 @@ def __init__(
                 >>> module_cls = MyModule
             args (Sequence, optional): args to be passed to ``module_cls``.
             kwargs (Dict, optional): kwargs to be passed to ``module_cls``.
+            module_rref (RRef[nn.Module], optional): If provided, no new module will be actually created,
```

Comment:
instead of adding a argument that's mutual exclusive to other args, should we just add a static method to the RemoteModule class, like `from_rref(RRef[nn.Module]) -> RemoteModule`? This way we can avoid the confusion of disregarding other args

Reply:
Yeah, I plan to provide an alternate API for this purpose. Basically the user can call `__new__` to create an empty instance, and then call a separate API to initialize from a `model_rref`.

Will do this in a separate RP.

Reply:
hmmm ok sounds good, but this do make the `RemoteModule` takes an `module_rref` as a arg and disregard the args/kwargs right? I think directly exposing this might not be good in the constructor, as user might use it and it's hard to remove it in the future (as we want to create a separate API to initialize from model_rref